applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -154,6 +154,68 @@ config BLE_ADV_TX_POWER_DBM
 	default -20 if BLE_ADV_TX_POWER_NEG_20DBM
 	default -40 if BLE_ADV_TX_POWER_NEG_40DBM
 
+config NRF_21540_ACTIVE
+	bool "Use front end module nRF 21540"
+	default n
+	help
+	  The front end module can help boost the TX power as high as 20 dBm.
+	  Usually set through buildprog, if not set there, remember to add
+	  -DSHIELD=nrf21540_ek_fwd when building app core
+
+config NRF_21540_MAIN_DBM
+	int
+	default 0 if NRF_21540_MAIN_TX_POWER_0DBM
+	default 10 if NRF_21540_MAIN_TX_POWER_10DBM
+	default 20 if NRF_21540_MAIN_TX_POWER_20DBM
+
+choice NRF_21540_MAIN_TX_POWER
+	prompt "TX power for the secondary channels"
+	default NRF_21540_MAIN_TX_POWER_10DBM
+	help
+	  Set the TX power for the secondary channels (0-36).
+	  Note that it is only allowed to set these to a maximum of +10dBm unless
```

Comment:
Suggest: 
	  
Note that it is special regulatory requirements needs to be met when using the front end module. 
If the nRF21540 is used with the nRF5340 Audio DK the actual output power will be about 25% lower
due to the VDD being 1.8V instead of the nominal 3.3V

Reply:
"Please check your local regulations for max output power"